static/js/app/cloudfoundry.js

```diff
@@ -0,0 +1,57 @@
+(function() {
+    // CloudFoundry Service
+    var cloudfoundry = function($http) {
+
+        // Get current authentication status from server
+        var getAuthStatus = function() {
+            return $http.get('/v2/authstatus')
+                .then(function(response) {
+                    return response.data.status;
+                }, function(response) {
+                    return response.data.status;
+                });
+        };
+
+        // Get organizations
+        var getOrgs = function() {
+            return $http.get('/v2/organizations')
+                .then(function(response) {
+                    return response.data.resources;
+                });
+        };
+        // Get organization spaces details
+        var getOrgSpaceDetails = function(org) {
```

Comment:
`org` should not be null because it's populated from [a list of orgs](https://github.com/18F/cf-console/blob/master/static/js/app/partials/dashboard.html#L20). Without an org the element that calls this function doesn't exist. Also, I think every org contains the elements requested.